test/utils/extractParts.js

```diff
@@ -8,12 +8,12 @@ module.exports = function (body, boundary) {
     .split(partRegexp)
     .forEach(function (part) {
       // Ignore empty strings in the array.
-      if (part.length === 0) return;
+      if (part.trim().length === 0) return;
```

Comment:
yes, not really sure about this. But the headers on request has some whitespaces between that's why I needed to trim them.

This is using request:

```
----------------------------454702286045209682268136
Content-Disposition: form-data; name="users"; filename="users.json"
Content-Type: application/json

[]

----------------------------454702286045209682268136
Content-Disposition: form-data; name="connection_id"

con_test
----------------------------454702286045209682268136--
```

and this superagent:

```
----------------------------392237490125139977580197Content-Disposition: form-data; name="connection_id"con_test----------------------------392237490125139977580197Content-Disposition: form-data; name="users"; filename="users.json"Content-Type: application/json[]----------------------------392237490125139977580197--
```

Do you think it can be an issue?